docs/content/intro/tutorial04.md

```diff
@@ -269,12 +269,3 @@ And we should now see the Checks tab with updates/results from the resulting Che
 This concludes the basic tutorial. If you are familiar with Brigade and are interested in learning how to refactor brigade.js into a more efficient test pipeline, check out [Advanced tutorial: Writing efficient pipelines][efficient-pipelines].
 
 You might also be scratching your head on what to [read next][readnext].
-
----
-
-Prev: [Part 3][part3] `|` Next: [Next Steps][readnext]
-
-[efficient-pipelines]: writing-efficient-pipelines.md
```

Comment:
Some or all of lines 277-280 need to stay, as long as they are referenced in-line above (lines 269 and 271 for example)